extensions/Worker.Extensions.ServiceBus/src/ServiceBusSessionMessageActionsConverter.cs

```diff
@@ -30,7 +30,8 @@ public ValueTask<ConversionResult> ConvertAsync(ConverterContext context)
             try
             {
                 var foundSessionId = context.FunctionContext.BindingContext.BindingData.TryGetValue("SessionId", out object? sessionId);
-                if (!foundSessionId)
+                var foundSessionIdArray = context.FunctionContext.BindingContext.BindingData.TryGetValue("SessionIdArray", out object? sessionIdArray);
```

Comment:
If we find SessionId, do we need to still look up SessionIdArray?

Reply:
Also, do we need to maintain these two stack bool variables? Couldn't we just resolve down to the sessionId here?

Reply:
makes sense! I created a method and just return the sessionId